### PR TITLE
Debian 9 support

### PR DESCRIPTION
**Pull request by lf94**
Support to build and use on Debian 9 (stretch) !

For those who are stuck on Debian 9 for awhile.

Preferable to be kept in a separate branch, since Debian 10 should work out of the box.

### CODE REVIEW

**Comment by lf94** [comment]
I have further simplified the installation process by including the built sdl2 packages.

Also fixed a codec finding bug.

I can confirm it's working here now :) So awesome! Fantastic work @thestr4ng3r !

File: gui/src/videodecoder.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 VideoDecoder::VideoDecoder(ChiakiLog *log) : log(log)
 {
+  avcodec_register_all();
```
Comment (thestr4ng3r) [inline gui/src/videodecoder.cpp]: This has been deprecated in avcodec 58.10.100, so it should be surrounded by an `#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(58, 10, 100)`.
Also, indentation must be tabs.

**Comment by lf94** [comment]
> Binaries should never be included in git, especially not if they are only specific to a single environment. In addition to that, I can not verify the integrity of those packages. So I will not have binaries in this repo, even on separate branches, sorry.
> 
> Please also share some details about why you would need to replace the cmake find modules. The current ones on master are fine in general and might only need a few small changes.

Of course not - but the point is anyone who is using debian 9 to run this, is most likely going to want to with minimal work. This is another reason to yet keep it on a separate branch.

The cmake find modules did not work properly on debian 9, that's the only reason. "Fine in general" was not good enough here.

I care about people *using* the software. I will close this PR and have people directed toward my fork then. Thank you regardless for chiaki. :)

**Comment by lf94** [comment]
> In addition to that, I can not verify the integrity of those packages.

And yes you can. Go build SDL2 2.0.6 for debian.

**Comment by thestr4ng3r** [comment]
> The cmake find modules did not work properly on debian 9, that's the only reason. "Fine in general" was not good enough here.

That is why I am asking what exactly the errors are in that case. Usually it needs only minor changes to get them compatible.

> And yes you can. Go build SDL2 2.0.6 for debian.

What I mean by integrity is that you can't easily check whether these packages actually were built from the unmodified SDL2 source. If you want to keep them in a fork, that if of course ok.

Feel free to open another pr with only the `avcodec_register_all()` change in an `#if`. I think that looks good.

**Pull request closed without merging**